build/build.ps1

```diff
@@ -21,7 +21,8 @@ Param(
 
 set-strictmode -version 2.0
 $ErrorActionPreference = "Stop"
-
+[Net.ServicePointManager]::SecurityProtocol = [Net.SecurityProtocolType]::Tls12
```

Comment:
Is there a doc link that explains the implications of this change?

Reply:
Other than https://msdn.microsoft.com/en-us/library/system.net.servicepointmanager.securityprotocol(v=vs.110).aspx?